pkg/network/node/ovs/ovs.go

```diff
@@ -409,8 +423,9 @@ func (ovsif *ovsExec) UpdateOVSMetrics() {
 
 // ovsExecTx implements ovs.Transaction and maintains current flow context
 type ovsExecTx struct {
-	ovsif *ovsExec
-	flows []string
+	ovsif  *ovsExec
+	flows  []string
+	groups []string
```

Comment:
There should be a single array, because order matters (ie, you can't add a flow that outputs to a group if the group doesn't exist yet). Call it `mods` (the man page talks about "flow mods" and "group mods").

Reply:
done I think I did this right